pom.xml

```diff
@@ -77,10 +77,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.dropwizard</groupId>
-            <artifactId>dropwizard-metrics</artifactId>
-            <version>0.9.2</version>
-            <scope>provided</scope>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
+            <version>3.1.2</version>
```

Comment:
I think I'd still like this to be `provided` scope so that this package is never going to override the metrics version declared elsewhere in a consuming project.

This will cause errors is somebody attempts to declare a dep on this package without declaring a dep on metrics-core, but that seems unlikely.

Do you have thoughts on that, @joschi?